plugin/helpers.go

```diff
@@ -10,6 +10,8 @@ type Helpers interface {
 	// Returns the id of the resulting bot.
 	EnsureBot(bot *model.Bot) (string, error)
 
+	CheckRequiredConfig(requiredConfig, actualConfig *model.Config) (bool, error)
```

Comment:
Let's add godoc here:
```suggestion
    // CheckRequiredConfig returns true if the actual configuration matches the non-nil parts of the required configuration.
	CheckRequiredConfig(requiredConfig, actualConfig *model.Config) (bool, error)
```